myst.yml

```diff
@@ -0,0 +1,29 @@
+# See docs at: https://mystmd.org/guide/frontmatter
+version: 1
+project:
+  id: 7a62a86d-a893-4ab1-9473-b1a957f78902
+  title: GTSAM Docs
+  description: GTSAM is a library of C++ classes that implement smoothing and mapping (SAM) in robotics and vision, using factor graphs and Bayes networks as the underlying computing paradigm rather than sparse matrices.
+  github: https://github.com/borglab/gtsam
+  toc:
```

Comment:
There are already various markdown files in the docs folder that we might also want to include.